app/src/main/java/org/jellyfin/androidtv/model/itemtypes/Episode.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.model.itemtypes
+
+import android.graphics.Bitmap
+
+data class Episode (
+	val id: String,
+	val canDelete: Boolean,
+	val canDownload: Boolean,
+	//TODO: Chapters: ArrayList<ChapterInfoDto>
+	val communityRating: Float,
```

Comment:
Should probably use a double

Reply:
BaseItemDto has a float though, should I still change it?

Reply:
Hmm... It doesn't really make sense to store a score as float IMO and I expect the new API to use doubles too so we can convert it for now

Reply:
Okay, will change it then